misc-utils/uuidd.c

```diff
@@ -526,20 +525,21 @@ static void server_loop(const char *socket_path, const char *pidfile_path,
 				num = 1000;
 			if (num * UUID_LEN > (int) (sizeof(reply_buf) - sizeof(num)))
 				num = (sizeof(reply_buf) - sizeof(num)) / UUID_LEN;
+			memcpy(reply_buf, &num, sizeof(num));
 			__uuid_generate_random((unsigned char *) reply_buf +
 					      sizeof(num), &num);
+			reply_len = sizeof(num) + (UUID_LEN * num);
```

Comment:
It does not seem more readable and robust. It seems (for reader) __uuid_generate_random() can modify `num` and it seems better to use it in memcpy() after __uuid_generate_random().

Reply:
The memcpy() is after __uuid_generate_random() in ea9416925c00cd3f364e05eb539aaba5bf7330c1 but before debug loop. I think it is better to write the number first to output buffer before payload that follows so that writes and data are in same order.